war/src/main/js/pluginSetupWizardGui.js

```diff
@@ -613,6 +616,44 @@ var createPluginSetupWizard = function() {
 			$c.slideDown();
 		}
 	};
+	
+	// call to change to setup security
+	var setupSecurity = function() {
+		setPanel(securityPanel, {
+			baseUrl: jenkins.baseUrl
+		}, function() {
+		    console.log('frame has been added to the page');
+			$('iframe').load(function() {
+				var location = $(this).contents().get(0).location.href;
+			    console.log('frame has (re)loaded, is showing: ' + location);
+			});
```

Comment:
I think it would be a good idea to explain in comments why this is using an `iframe`.